src/request.toit

```diff
@@ -10,43 +10,73 @@ import .chunked
 import .response
 import .connection
 
-class Request:
+/// Legacy interface for older code.
```

Comment:
In theory we could also just go to a new version number, but since this is easy enough...

Should this interface then be marked as deprecated?